test/packages/file_upload_example/file_upload/files_data/fields/fields.yml

```
@@ -0,0 +1,180 @@
+- name: agent_id
+  type: keyword
+  description: agent id
+- name: action_id
+  type: keyword
+  description: action id
+- name: source
+  type: keyword
+  description: source
+- name: file
+  title: File
+  type: group
+  fields:
+    - name: Status
+      type: keyword
+      description: status
+    - name: ChunkSize
+      type: integer
+      description: chunk size
+    - name: Compression
+      type: keyword
+      description: compression type
```

Nit. Different casing used here an in other parts of the mapping.
```suggestion
    - name: status
      type: keyword
      description: status
    - name: chunk_size
      type: integer
      description: chunk size
    - name: compression
      type: keyword
      description: compression type
```

---

to follow ECS patterns: lowercase fields represent ECS standard.

ECS is explicitly lowercase only and encourages initial capitals to indicate custom extensions (i.e. fields that are not ECS)

---

@pzl I was basing the above on your working set of mappings/indices.  Let me know if there's any issue with the naming above.